public-images/public-images-how-to/run-a-vagrant-box.rst

```diff
@@ -2,7 +2,25 @@
 
 Run a Vagrant box
 =====================
-The quickest way to get a vagrant environment up and running is ``vagrant init <BOX>``. For upstream, use a call such as ``vagrant init ubuntu/focal64``. To bring up the image, ``vagrant up``. This will create a VM, create an ssh key and associate it with the vagrant user, and mount the default synced directory (``./`` to ``/vagrant``). The image should boot relatively quickly, dependent on hardware. The default Vagrant timeout is five minutes, and if it's even close to that, something has gone wrong.
+The quickest way to get a vagrant environment up and running is to run:
+
+.. code::
+
+   vagrant init <BOX>
+
+For upstream, use a call such as:
+
+.. code::
+
+   vagrant init ubuntu/focal64
```

Comment:
As per [John's previous comment](https://github.com/canonical/ubuntu-cloud-docs/pull/214#discussion_r1569235125), lets point to `ubuntu/jammy64`. Something that got missed when this was all one paragraph.

Reply:
Done, thanks!